examples/producer.py

```diff
@@ -19,22 +19,55 @@
 # Example Kafka Producer.
 # Reads lines from stdin and sends to Kafka.
 #
-
 from confluent_kafka import Producer
 import sys
+import getopt
```

Comment:
I think we only the stats_cb example in of the examples (to keep the examples small).
Remove either this one or the other one.